src/include/sof/audio/component.h

```diff
@@ -303,11 +391,6 @@ struct comp_driver_info {
 	struct list_item list;		/**< list of component drivers */
 };
 
-/** \brief Holds list of registered components' drivers */
-struct comp_driver_list {
-	struct list_item list;	/**< list of component drivers */
-};
-
```

Comment:
Why is this removed from here. I would think that this is a basic structure that needs to be here.

Reply:
This structure is used by the infrastructure only, in a single place. Component developers do not need to know it exists imo.

Reply:
Ok.